server.js

```diff
@@ -15,10 +15,95 @@ const app = express();
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+///
+const ThoughtSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    default: "unknown",
+    //to delete the whitespace
+    trim: true,
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true,
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date(),
+  },
+});
 
+const Thoughts = mongoose.model("Thoughts", ThoughtSchema);
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const WelcomePage = {
+    Hello: "Welcome to Happy Thoughts API",
+    Routes: [
+      {
+        "/thoughts": "Get the thoughts added",
+      },
+    ],
+  };
+  res.send(WelcomePage);
+});
+app.get("/thoughts", async (req, res) => {
+  try {
+    const thoughts = await Thoughts.find()
+      .sort({ createdAt: "desc" })
+      .limit(20)
+      .exec();
+    res.status(200).json(thoughts);
+  } catch (err) {
+    res.status(400).json({
+      success: false,
+      response: "Bad request, couldnot fetch thoughts.",
+    });
+  }
+});
+//POST request
+app.post("/thoughts", async (req, res) => {
+  const { message } = req.body;
+  const createdAt = new Date(
+    new Date(Date.now()).toLocaleString("sv-SE", {
+      timeZone: "Europe/Stockholm",
+    })
```

Comment:
Time looks accurate in the backend, although (and I had the same problem when using timeZone Europe) time is wrong when looking at the netlify page (new messages are told to be posted about two hours ago). New added posts therefore shows up in the middle of all posts instead of at the top.